packages/grid/x-grid/src/XGrid.test.tsx

```diff
@@ -209,17 +209,17 @@ describe('<XGrid />', () => {
 
       const row = document.querySelector('[role="row"][aria-rowindex="2"]');
       const checkbox = row!.querySelector('input');
-      expect(row).to.not.have.class('Mui-selected');
+      expect(row).to.not.have.class('Mui-selected', 'class mui-selected 0');
       expect(checkbox).to.have.property('checked', false);
 
       fireEvent.click(screen.getByRole('cell', { name: 'Nike' }));
-      await raf();
-      expect(row!.classList.contains('Mui-selected')).to.equal(true);
+      await sleep(100)
+      expect(row!.classList.contains('Mui-selected')).to.equal(true, 'class mui-selected 1');
```

Comment:
```suggestion
      expect(row!.classList.contains('Mui-selected')).to.equal(true);
```

Reply:
By leaving this we know where the test can potentially failed. Up to you?